zcash_client_sqlite/src/wallet/sapling.rs

```diff
@@ -990,6 +1027,11 @@ pub(crate) mod tests {
         let addr2 = extsk2.default_address().1;
         let to = addr2.into();
 
+        // TODO: This test was originally written to use the pre-zip-313 fee rule
+        // and has not yet been updated.
```

Comment:
Please file an issue to update these tests, if there isn't one already.

Reply:
#1025